src/cmd/manage_nsfs.js

```diff
@@ -95,6 +95,7 @@ async function main(argv = minimist(process.argv.slice(2))) {
         const user_input = user_input_from_file || argv;
         config_root = argv.config_root ? String(argv.config_root) : config.NSFS_NC_CONF_DIR;
         if (!config_root) throw_cli_error(ManageCLIError.MissingConfigDirPath);
+        if (argv.config_root) config.NSFS_NC_CONF_DIR = String(argv.config_root);
```

Comment:
the argv.config_root? 
I would love to remove it, it's very annoying

Reply:
I meant to the option of passing config_root, 
I added it because if we won't update it the master_keys.json will get written to /etc/noobaa.conf.d/

Reply:
why would it be created always in etc? our config.js code already handles config_dir_redirect no?

Reply:
yes, but this was initially introduced before we had anything, and for cases when there is not config_dir_redirect.
I think we use it only for tests